HTTP_API.md

```diff
@@ -1169,3 +1169,58 @@
     ```
     $ curl --form "template=@Foo.jfc" localhost:8181/api/v1/templates
     ```
+
+## V2 API
+
+### Quick Reference
+
+| What you want to do                                                       | Which handler you should use                                                |
+| ------------------------------------------------------------------------- | --------------------------------------------------------------------------- |
+| **Recordings in target JVMs**                                             |                                                                             |
+| Create a snapshot recording in a target JVM                               | [`TargetSnapshotPostHandler`](#TargetSnapshotPostHandler)                   |
```

Comment:
>This breaks the anchor links in HTTP_API.md since there are now two handlers by the same name, performing generally the same role, but for API V1/V2. 

I tried clicking on the anchor button next to the header and it looks like github adds a counter for duplicate names. So I tested it using `[...](#TargetSnapshotPostHandler-1)` and it looks like that works.

>IMO the document should probably be split per version to avoid problems like this and prevent it from growing indefinitely.

Yeah, I think that's a good idea too. If V2 had the same amount of handlers as V1, I think that would make the document really hard to use and navigate for whichever version was at the bottom.

Reply:
re: anchor counts - cool, that's very neat

re: doc splitting - alright, that's probably a topic for a separate PR. I'll leave it as-is tagged on the end for now and then we can figure out the best way to split it up